odoo/modules/module.py

```diff
@@ -6,15 +6,17 @@
 import copy
 import functools
 import importlib
+import importlib.metadata
 import logging
 import os
-import pkg_resources
```

Comment:
It is also used in `setup/requirements-check.py`, can you replace it there too?

Reply:
> It is also used in `setup/requirements-check.py`, can you replace it there too?

@kmagusiak  I did something. I'm not entirely sure what that script is supposed to do precisely, so please check it still does what you expect.